### PR TITLE
SikuliX: Add version 2.0.5

### PR DESCRIPTION
**Pull request by jhwohlgemuth**
This PR adds support for installing [SikuliX](https://sikulix.github.io/), a really cool UI-based robotic process automation tool. SikuliX makes it really easy to automate just about anything on a computer. My thought is that it would get more usage if it was easier to install. As of today, all you need to do is download a JAR file and run it with Java, but who has time for that? I have wanted to do scoop install sikulix for years. Hopefully my dream will come true soon...

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/sikulix.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            ]
+        }
+    },
+    "checkver": {
+        "url": "https://raiman.github.io/SikuliX1/downloads.html",
+        "regex": "Version \\d[.]\\d[.]\\d (latest stable version)"
+    },
+    "autoupdate": {
+        "url": "https://launchpad.net/sikuli/sikulix/$version/+download/sikulixide-$version.jar"
+    }
```
Comment (ftk) [inline bucket/sikulix.json:21]:
```suggestion
    "checkver": {
        "url": "https://raiman.github.io/SikuliX1/downloads.html",
        "regex": "Version ([\\d.]+) \\(latest stable version\\)"
    },
    "autoupdate": {
        "url": "https://launchpad.net/sikuli/sikulix/$version/+download/sikulixide-$version.jar",
        "bin": [
            [
                "java.exe",
                "sikulix",
                "-jar",
                "\"$dir\\sikulixide-$version.jar\""
            ]
        ]
    }
```
BTW you can test autoupdate by typing
```pwsh
& "$Env:SCOOP\apps\scoop\current\bin\checkver.ps1" -Dir . -ForceUpdate sikulix
```

Comment (jhwohlgemuth) [reply bucket/sikulix.json]: @ftk  Awesome! Thank you so much! I will test your suggested change with your suggested method as soon as I can. Thanks again.

**Comment by jhwohlgemuth** [comment]
No rush, but I am pretty sure this PR is ready to go...I am looking forward to being able to install SikuliX with just `scoop install sikulix` 🤓 

If this PR is still lacking something, I will endeavor to address any issues 🧐

**Comment by jhwohlgemuth** [comment]
@ftk or anyone, is there anything else I can do on this? It would be really fun to be able to `scoop install sikulix`...any help would be appreciated 😃

**Review by HUMORCE — APPROVED** [review]
Tested.

**Pull request merged**